skelebot/components/environments.py

```diff
@@ -0,0 +1,52 @@
+"""Environments Component"""
+
+import os
+import re
+from schema import Schema, And, Optional
+from ..objects.component import Activation, Component
+from ..systems.execution import docker
+
+HELP_MESSAGE = "Display the available environments for the project"
+
+class Environments(Component):
+    """
+    Envs Class
+
+    Provides the ability to inspect the available environments for the project.
+    """
+
+    activation = Activation.PROJECT
+    commands = ["envs"]
+
+    def addParsers(self, subparsers):
+        """
+        SkeleParser Hook
+
+        Adds a parser for the jupter command the starts up the notebooks in Docker
+        """
+
+        subparsers.add_parser("envs", help=HELP_MESSAGE)
+        return subparsers
+
+    def execute(self, config, args, host=None):
+        """
+        Execution Hook
+
+        Executed when the envs command is provided it reads the skelebot files from the local
+        project to determine the list of available environments.
+        """
+
+        envs = ["[default]"]
+
+        # Read envs from disk
+        for fl in os.listdir():
+            if re.match("skelebot-\\w+[.]yaml", fl):
```

Comment:
This regex isn't capturing filenames with multiple dashes like `skelebot-train-score.yaml`. Unless that is not considered a valid environment file:
![Screen Shot 2020-10-27 at 9 29 21 AM](https://user-images.githubusercontent.com/13685125/97308218-03be4080-1837-11eb-9b4f-074e5ab14182.png)

Reply:
I think in the docs we use the format skelebot-[env].yaml, but it's true we don't say what characters can't be used in environment names. We can either make the env names more strict, or open up the regex to more characters. I prefer the idea of env names being simple (just alphabet characters) since they have to be passed as parameter values to the skelebot command.

Reply:
That makes sense. Simple can be better!